CONTRIBUTING.md

```diff
@@ -8,22 +8,33 @@ See [Superchain Upgrades] OP-Stack specifications.
 
 ## Adding a chain
 
-### Adding a standard chain
-
-#### 1. Set env vars
+### 1. Set env vars
 
 To contribute a standard OP-Stack chain configuration, the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local monorepo folder.
 
 First, make a copy of `.env.example` named `.env`, and alter the variables to appropriate values.
 
-#### 2. Run script
-Then, run
+### 2. Run script
+#### Standard chains
+If your chain meets the definition of a **standard** chain, you can run
```

Comment:
Guessing we don't have a public canonical definition of standard right now but it might be useful to link out to that here if we do.

Reply:
https://github.com/ethereum-optimism/superchain-registry/issues/142